contracts/interfaces/ITokenManager.sol

```diff
@@ -112,4 +60,18 @@ interface ITokenManager is IBaseTokenManager, IOperatable, IFlowLimit, IImplemen
      * @param flowLimit_ The maximum difference between the tokens flowing in and/or out at any given interval of time (6h).
      */
     function setFlowLimit(uint256 flowLimit_) external;
+
+    /**
+     * @notice A function to renew approval to the service if we need to.
+     */
+    function addServiceApproval() external;
+
+    /**
+     * @notice Getter function for the parameters of a lock/unlock TokenManager.
+     * @dev This function will be mainly used by frontends.
+     * @param operator_ The operator of the TokenManager.
+     * @param tokenAddress_ The token to be managed.
+     * @return params_ The resulting params to be passed to custom TokenManager deployments.
+     */
+    function params(bytes calldata operator_, address tokenAddress_) external pure returns (bytes memory params_);
```

Comment:
We may change the params encoding, in which case this function would need to be updated. I think it's fine to not have this

Reply:
i like giving frontends the ability to figure out params by querying the chain.